modules/webstyle/lib/webstyle_templates.py

```diff
@@ -939,7 +945,7 @@ def detailed_record_container_bottom(self, recid, tabs, ln=CFG_SITE_LANG,
     <br/>
     """ % {'similar' : similar,
            'dates' : creationdate and '<div class="recordlastmodifiedbox" style="position:relative;margin-left:1px">&nbsp;%(dates)s</div>' % {
-                'dates': _("Record created %(x_date_creation)s, last modified %(x_date_modification)s") % \
+                'dates': _("Record added %(x_date_creation)s, last modified %(x_date_modification)s") % \
                 {'x_date_creation': creationdate,
```

Comment:
Some nomenclature musings.  If we have "datecreated", "datemodified", and "earliestdate" dates, then I'm afraid things may not be fully clear what the latter means, based on its sole name.  Partially perhaps because one can say "earliest modification date" which is not the same as the creation date, or "earliest citation date" which would be the date of the first citation, or "earliest publication date" in case a work was re-published, or "earliest X date" in general.  Seems like the qualifier X is missing, as it were.

It may be worth pondering whether we shouldn't find better name.  And, perhaps, whether we actually *need* a new DB column for storing it?  Wouldn't it be sufficient to have the above BibCheck rule running periodically to force some designed new metadata field, say 260/269 or something, to be equal to the earliest publication date?  For example, do we want to expose this information to cataloguers, or is it just a "derived field" living in the database for convenience?  Are we getting more derived fields of this kind, such as "latest refextract date", "latest keywording date", as we mused elsewhere could be convenient for internal processing?